app/javascript/invoker.js

```diff
@@ -0,0 +1,39 @@
+import headers from './headers'
+import lifecycle from './lifecycle'
+import state from './state'
+import urls from './urls'
+import { dispatch } from './events'
+import { render } from './renderer'
+
+const parseError = error => {
+  const errorMessage = `Unexpected error performing a TurboBoost Command! ${error.message}`
+  dispatch(lifecycle.events.clientError, document, { detail: { error: errorMessage } }, true)
+}
+
+const parseResponse = response => {
```

Comment:
The method name doesn't make clear that there's a side effect going on here.

```suggestion
const parseAndRenderResponse = response => {
```